src/Countries/US/NorthCarolina/NorthCarolinaIncome/V20190101/NorthCarolinaIncome.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\NorthCarolina\NorthCarolinaIncome\V20190101;
+
+use Appleton\Taxes\Classes\Payroll;
+use Appleton\Taxes\Countries\US\NorthCarolina\NorthCarolinaIncome\NorthCarolinaIncome as BaseNorthCarolinaIncome;
+use Appleton\Taxes\Models\Countries\US\NorthCarolina\NorthCarolinaIncomeTaxInformation;
+
+class NorthCarolinaIncome extends BaseNorthCarolinaIncome
+{
+    const SUPPLEMENTAL_TAX_RATE = 0.05599;
+
+    const TAX_RATE = 0.0535;
```

Comment:
Whats the diff between the 2 tax rates? It looks like supplemental has the old 5.599 rate and tax_rate had a different one?